verify_test_type/README.md

```diff
@@ -0,0 +1,43 @@
+<h1 align="center">Verify Test Type</h1>
+
+<p align="center">Script Purpose: Ensure the identification of the test type (e.g., ID, OD, NOD, etc.) for new tests discovered in a repository. </p>
+
+## Steps to Run the Script locally
+
+#### 1. Go to the root directory
+
+Go to the root directory of the repository which has the newly found flaky test.
+
+#### 2. Download the script in your repository
+
+Ensure you have wget installed on your system. You can install it using the package manager of your operating system:
+- For Ubuntu/Debian: sudo apt-get install wget
+- For macOS (using Homebrew): brew install wget
+- For Windows: Download it from GNU Wget.
+
+Now download the script using this command:
+
+```
+wget https://raw.githubusercontent.com/TestingResearchIllinois/idoft/main/verify_test_type/verify_test_type.sh
+```
+
+#### 3. Make the script executable
+
+Use this command to make the script executable:
```

Comment:
Wouldn't need this for `git clone` (in fact, shouldn't need it if the file has the right permission in the repo, and if the file doesn't have, just open a PR for that).